app/controllers/projects_controller.rb

```diff
@@ -31,24 +32,68 @@ def update
     @project.assign_attributes(project_params)
     maintain_history("Project", @project) unless @project.changes.empty?
     if @project.save
-      flash[:success] = 'Category updated successfully...'
+      flash[:success] = 'Project updated successfully...'
       redirect_to projects_path
     else
-      flash.now[:alert] = 'Category not updated successfully...'
+      flash.now[:alert] = 'Project not updated successfully...'
       render 'edit'
     end
   end
 
   def destroy
     if @project.destroy
       delete_history("Project", @project)
-      flash[:alert] = 'Category deleted successfully...'
+      flash[:alert] = 'Project deleted successfully...'
     else
-      flash[:notice] = 'Category not deleted successfully...'
+      flash[:notice] = 'Project not deleted successfully...'
     end
     redirect_to projects_path
   end
 
+  def developers_list
+    @developers = User.developer
+  end
+
+  def QAs_list
+    @QAs = User.QA
+  end
```

Comment:
why we need this action?

Reply:
to display QA list available for assignment